### ISSUE TITLE
Cpu heating issue while using SPMC. 

### ISSUE THREAD

**Issue by Pretxt** [koying/SPMC#955]
Have tried a few apps like Netflix and my cpu is around 60C But when using SPMC it lands at 80-90. Is there anything I can do? I'm looking into adding a fan and stuff. But if there is some setting  I have missed.

**Comment by Pretxt** [comment]
I use h96pro+ forgot to add that.

**Comment by TinderboxUK** [comment]
My S905X - Android v6 SPMC runs at least 10c hotter than MX Player.

**Comment by Pretxt** [comment]
I had even more issued with the latest kodi :/

**Comment by wrxtasy** [comment]
lets see it could be:

(a) Poor device thermal engineering with an inadequate heat sink used.
(b) Poor Android Firmware with too many un-neccessary background processes running and those not being properly managed at the buggy OS level.
(c) Poorly written drivers with inadequate testing done before device sale.
(c) Trying to play video files that cannot be hardware decoded so Software decoding is used and that is stressing the CPU's and then leading to problem (a) again.

TIP: attach a Keyboard and press O - if you see ff-mpeg anything video is being software decoded.

I would also go to the Freaktab website and search and install bug busted Android Firmware if you want to see any improvements at all. No fan will likely be needed then ;-)

**Comment by Pretxt** [comment]
Thank you for your response! I will walk this through! Much appreciated!

**Comment by koying** [comment]
Obviously, first thing to do before comparing Kodi/SPMC to any "plain" player is to start from scratch, without *any* addons.
Those suck CPU big time ;)

**Comment by Pretxt** [comment]
I'm having the issues while using TVheadend HTSP Client. Works very well on my phone and pc but not on the h92pro+ ain't buying anymore stuff from TomTop, wish and those sites lol. But I will look into you tips!
